mock-core-configs/etc/mock/templates/almalinux-8.tpl

```diff
@@ -65,7 +65,7 @@ gpgkey=file:///usr/share/distribution-gpg-keys/alma/RPM-GPG-KEY-AlmaLinux
 fastestmirror=1
 
 [devel]
-name=AlmaLinux $releasever - Devel (WARNING: UNSUPPORTED - FOR BUILDROOT USE ONLY!)
```

Comment:
I think we should synchronize with `WARNING:` and note it's all unsupported.

Reply:
Can you spell the exact wording you prefer?  I think it is OK now, but I'll accept your variant.  I just don't want to spend too much time on this PR.

Reply:
`(WARNING: UNSUPPORTED - FOR BUILDROOT USE ONLY!)`

Reply:
For EPEL, the `local` repos are mainly pre-compose (direct koji access), so the unsupported/warning doesn't apply there.

Reply:
Hmm, strictly speaking, is it different?  Those packages are neither supported ..

Reply:
I mean, it contains all sort of buildoverrides, and are not signed, and anything may disappear.

Reply:
Fair point, actually. But generally, the expectation is that Koji stuff eventually ships out for EPEL.

Reply:
Ack, and "local" repos are opt-in.

Reply:
Yup!

Reply:
Ouch, the Devel repo is opt-in intentionally?

Reply:
Ok, according to the latest info those repos are mostly empty anyway.

Reply:
For CentOS, yes, because that's all moved to `local`, but for Alma/Rocky, they're full of everything that's normally in `local`.

Reply:
This is off-topic, but I'm curious why ...?  Devel (mirrored) is orthogonal thing to 'local'?